reactor-core/src/test/java/reactor/core/publisher/MonoJustTest.java

```diff
@@ -50,13 +50,26 @@ public void normal() {
 	                .verifyComplete();
     }
 
+	@Test
```

Comment:
this is not needed. `Mono.just` emits whatever value is passed to it (`null` being an invalid value) without interpreting the value in any way. the optional could be a `String` or anything. The second test you added makes a little bit of sense though, although technically `Optional.empty()` (tested further down) and `Optional.ofNullable(null)` are the same.

Reply:
Hi @simonbasle, I have just removed the first test.